py-scoring/build.gradle

```diff
@@ -217,10 +205,10 @@ task cleanGeneratedApi(type: Delete) {
   delete "$projectDir/src-gen"
 }
 
-task generateMOJOApi(type: JavaExec, dependsOn: [cleanGeneratedApi, ':sparkling-water-api-generation:build']) {
+task generateMOJOApi(type: JavaExec, dependsOn: [cleanGeneratedApi, ':sparkling-water-assembly:shadowJar']) {
```

Comment:
Is this possible? the generated classes will be part of the final assembly jar. This looks like a cyclic dependency.